src/js/pyodide.ts

```diff
@@ -346,6 +345,8 @@ If you updated the Pyodide version, make sure you also updated the 'indexURL' pa
     throw new Error("Didn't expect to load any more file_packager files!");
   };
 
+  setHomeDirectory(Module, config.homedir);
```

Comment:
Why did you move `setHomeDirectory` after initializing the module? It's better if possible to keep all of this stuff before initializing Python because there are code paths related to virtual environments that care about these paths on initialization.

Reply:
There is a bug in main branch: `/home/pyodide/lib/python310.zip` is in the sys.path not `/lib/python310.zip`.

```
>>> import sys
>>> sys.path
['/home/pyodide', '/home/pyodide/lib/python310.zip', '/lib/python3.10', '/lib/python3.10/lib-dynload', '/lib/python3.10/site-packages']
```

moving `setHomeDirectory` after the initialization fixed this, but maybe there can be a better fix.